stable/redis-ha/templates/redis-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    redis-node: "true"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
```

Comment:
Again, I don't know about best practice for helm. We'll probably want to configure a cluster to increase slaves at some point so we can use them as read-replicas. I don't know that changing the number of master replicas would matter if it's just used to bootstrap. But if I use the master pod to boostrap and then remove it (as the README suggests), I'd want to make sure there were at least 2 redis nodes and 3 sentinel nodes still running.